### DISCUSSION TITLE
[Question] Any ETA for the next release ?

### DISCUSSION THREAD

**Discussion by rbioteau** [de-jcup/eclipse-asciidoctor-editor#411] [Q&A] [answered]
Hi @de-jcup , 

Do you have a planned timeline for the next release ? I'd like to integrate the latest changes in my RCP and it will be helpful to have a little bit of vision.

Thanks !

PS: If there are some topics/issues that block the release don't hesitate to point them out. I'll be glad to have look and help.

**Comment by rbioteau** [comment, marked as answer]
I'm an animal :see_no_evil: 
Already released: https://github.com/de-jcup/eclipse-asciidoctor-editor/releases/tag/v2.5.0

**Reply by de-jcup** [reply]
@rbioteau : I released some hours ago V2.5.1 because there was a serious bug with 2.5.0 when using MS Windows. Maybe this is also important for you (I think your are using Windows ? ) . See https://github.com/de-jcup/eclipse-asciidoctor-editor/milestone/33?closed=1 (or directly https://github.com/de-jcup/eclipse-asciidoctor-editor/issues/414)  for details.
